editions/multiwikiserver/tiddlers/$__StoryList_1.tid

```diff
@@ -0,0 +1,2 @@
+list: GettingStarted
+title: $:/StoryList
```

Comment:
I don't like nodejs wiki creating `$__StoryList_1.tid` while I already git itnore the `$__StoryList.tid`. I ignore this because it cause too much git backup revision.

Reply:
If it creates StoryList_1.tid, there is a bug in "file locking" mechanism. If the server is restarted and StroyList is still locked somehow, it will create the _1.tid file. 

**Instead of a work-around you should try to fix the underlaying problem**